src/SmartComponents/Portfolio/Portfolio.js

```diff
@@ -41,6 +45,36 @@ class Portfolio extends Component {
     }
   }
 
+  startRemovingProducts = () => {
```

Comment:
you can do these as single line methods:
```javascript
finishRemovingProducts = () => this.setState({ isRemoving: false });
```
Also applies to `finishRemovingProducts`